locales/hi_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -123,54 +123,54 @@ msgstr ""
 #. In the new DDG website, this token appears as the Tab name for Instant Answers that give general information about a subject. eg. https://duckduckgo.com/?q=Sony
 #. It is not "About DuckDuckGo".
 msgid "About"
-msgstr "के बारे में"
+msgstr "इसके बारे में"
 
 #. This is the side bar "About" link
 msgctxt "static_page"
 msgid "About"
-msgstr ""
+msgstr "डकडकगो के बारे में"
```

Comment:
`Duckduckgo` term will not be translated as it is a Trademark. Please leave it as it is.